examples/cursor_grab.rs

```diff
@@ -23,19 +24,23 @@ fn main() {
             Event::WindowEvent { event, .. } => match event {
                 WindowEvent::CloseRequested => *control_flow = ControlFlow::Exit,
                 WindowEvent::KeyboardInput {
-                    input:
-                        KeyboardInput {
+                    event:
+                        KeyEvent {
+                            logical_key: key,
                             state: ElementState::Released,
-                            virtual_keycode: Some(key),
                             ..
                         },
                     ..
                 } => {
-                    use winit::event::VirtualKeyCode::*;
+                    // WARNING: Consider using `key_without_modifers()` if available on your platform.
+                    // See the `key_binding` example
                     match key {
-                        Escape => *control_flow = ControlFlow::Exit,
-                        G => window.set_cursor_grab(!modifiers.shift()).unwrap(),
-                        H => window.set_cursor_visible(modifiers.shift()),
+                        Key::Escape => *control_flow = ControlFlow::Exit,
+                        Key::Character(ch) => match ch.to_lowercase().as_str() {
```

Comment:
Remind me why we need `to_lowercase` here? This is just an oversight?

Reply:
Because this example need to be compatible with the Web, so we are using `logical_key`, but `logical_key` is sensitive to <kbd>Shift</kbd> while the chosen action depends on wheter <kbd>Shift</kbd> is held down. Using `to_lowercase` is one of the many approaches to make this work. If this description is a bit confusing, try running the example without `to_lowercase`.

Reply:
Additionally, `to_lowercase` seems to have fewer cases where single-codepoint characters turn into multi-codepoint characters compared to `to_uppercase` ([playground exapmple](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=8efbe463775bb5f5c5f19d8b715422c0)).